src/rviz_visual_tools.cpp

```diff
@@ -1077,6 +1077,47 @@ bool RvizVisualTools::publishPolygon(const geometry_msgs::Polygon &polygon, cons
   publishPath(points, color, scale, ns);
 }
 
+  bool RvizVisualTools::publishWireframeCuboid(const Eigen::Vector3d &position,
+                                               const Eigen::Matrix3d &rotation_matrix,
+                                               const Eigen::Vector3d &min_point,
+                                               const Eigen::Vector3d &max_point,
+                                               const RvizVisualTools::colors color=RvizVisualTools::BLUE)
```

Comment:
Makes sense, do you know of a good eigen tutorial/how to combine rotation matrix & translation into `Affine3d`?